tests/suite_test.go

```diff
@@ -33,13 +37,18 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
+	opv1a1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
 	mcgopenshiftiov1alpha1 "github.com/red-hat-storage/mcg-osd-deployer/api/v1alpha1"
 	//+kubebuilder:scaffold:imports
 )
 
 // These tests use Ginkgo (BDD-style Go testing framework). Refer to
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
+const (
+	Namespace = "redhat-data-federation"
```

Comment:
```suggestion
	namespace = "redhat-data-federation"
```

No need to export the namespace. This usage will always be under the `tests` package.